launch/interaction/picovoice.launch

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<launch>
+    <!-- Get the machine file -->
+    <arg name="machine" default="localhost"/>
+
+    <arg name="robot_name"/>
+    <arg name="picovoice_robot_config" default="$(env HOME)/MEGA/credentials/picovoice/access_key_$(arg robot_name).yaml" />
```

Comment:
This appears to be an access key. if so also name it as such

Reply:
Leave it config matches telegram launch file. And it is a config as it sets ros parameters. Therefore it could configure anything.